.github/workflows/dev-ci.yml

```diff
@@ -98,7 +97,7 @@ jobs:
           }
           - {
             name: windows,
-            os: windows-2019,
+            os: windows-2022,
```

Comment:
Yes. I read in the doc that Visual study redistribuable (so microsoft compiler runtime) as to be 2022 one and not anymore 2019. To be sure that we absolutely need to use 2022 I'm going to revert this part in the PR and test what happen.
The impact for the user is that they will have to install it https://learn.microsoft.com/fr-fr/cpp/windows/latest-supported-vc-redist?view=msvc-170 if not already done (many applications rely on it)

Reply:
Confirmed, 2022 is needed
CUSTOMBUILD : error : On Windows, GraalVM Native Image for JDK 17 requires Visual Studio 2022 version 17.1.0 or later (C/C++ Optimizing Compiler Version 19.31 or later). [D:\a\pypowsybl\pypowsybl\build\temp.win-amd64-3.8\Release\native-image.vcxproj]
  Compiler info detected: cl.exe (microsoft, x64, 19.29.30148)
CUSTOMBUILD : error : To prevent native-toolchain checking provide command-line option -H:-CheckToolchain [D:\a\pypowsybl\pypowsybl\build\temp.win-amd64-3.8\Release\native-image.vcxproj]
CUSTOMBUILD : error : Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception [D:\a\pypowsybl\pypowsybl\build\temp.win-amd64-3.8\Release\native-image.vcxproj]